GoogleDataTransport/GDTCORLibrary/GDTCORStorage.m

```diff
@@ -216,6 +216,10 @@ - (void)addEventToTrackingCollections:(GDTCORStoredEvent *)event {
 #pragma mark - GDTCORLifecycleProtocol
 
 - (void)appWillForeground:(GDTCORApplication *)app {
+  dispatch_async(_storageQueue, ^{
+    self.runningInBackground = NO;
```

Comment:
Would it make sense to move the logic into the class invoking `GDTCORLifecycleProtocol` methods so the flag is managed in a single place and clients can just read it?

Reply:
This is what needs to be done. You can take a shot at it if you want, or wait until I'm back from vacation.